model/src/main/java/com/ge/predix/acs/model/ObligationType.java

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+ * Copyright 2017 General Electric Company
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+package com.ge.predix.acs.model;
+
+/**
+ *
+ * @author Sebastian Torres Brown
+ * 
+ * enum class to define the types of Obligations that are supported
+ * 
+ */
+
+public enum ObligationType {
```

Comment:
we could add this, if and when we add other types.  Especially because we don't see any use-case for it.

Reply:
we will make this not mandatory, defaulting to custom

Reply:
👍